starter-code/js/index.js

```diff
@@ -35,7 +84,12 @@ function createNewItemRow(itemName, itemUnitPrice){
 }
 
 function createNewItem(){
+  // var newItem = document.getElementsByClassName('product-wrapper');
```

Comment:
basically every time you use the create button to create a new item you will go through the create function for each element needed for the product your creating.